analyzers/nodejs/testdata/transitive-deps/npm-ls-json.json

```diff
@@ -0,0 +1,23 @@
+{
+  "dependencies": {
+    "a": {
+      "version": "1.0",
```

Comment:
[nit] standard semver has three fields: <major>.<minor>.<patch>. I don't think `npm` or `nodejs` require, expect, or enforce this, so it should be okay

Reply:
I'll change that, good call.